tide/wall/DataProvider.h

```diff
@@ -137,6 +140,11 @@ public slots:
 
     template <typename DataSources>
     void _updateTiles(DataSources& dataSources);
+
+    template <typename DataSource, typename URIType>
```

Comment:
The dataSources can have QString (for Pixelstreams) and QUuid (all the others) as an identifier.